external/kafka-0-10-sql/src/main/scala/org/apache/spark/sql/kafka010/KafkaMicroBatchStream.scala

```diff
@@ -23,16 +23,17 @@ import java.util.Optional
 import scala.collection.JavaConverters._
 
 import org.apache.kafka.common.TopicPartition
```

Comment:
Looks like style of imports is broken. You can run the Scala style checker via `dev/scalastyle` and see the violations.